src/main/java/tc/oc/pgm/rotation/Rotation.java

```diff
@@ -0,0 +1,130 @@
+package tc.oc.pgm.rotation;
+
+import com.google.common.base.Optional;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.bukkit.configuration.ConfigurationSection;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.map.PGMMap;
+
+public class Rotation implements PGMMapOrderProvider {
```

Comment:
How about this:
`PGMMapOrderProvider` -> `Rotation`
`Rotation` -> `SingleRotation implements Rotation`
`RotationManger` -> `RotationManager implements Rotation`

Reply:
I'll follow this scheme and apply it for the next update I push, and I'll also document it hopefully, so the comment above should be resolved by then.

Reply:
Ok with `PGMMapOrder`, `FixedPGMMapOrder`, and `PGMMapOrderManager` @BGMP

Reply:
Done. https://github.com/Electroid/PGM/pull/114/commits/fe83e3a18ec9419e8184475a0ccd8c4fb5a5e93a